templates/etc/fastd/fastd-blacklist.sh.erb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+PEER_KEY=$1
+
+if /bin/grep -Fq $PEER_KEY /etc/fastd/<%= @mesh_code %>-mesh-vpn/fastd-blacklist.json; then
```

Comment:
regarding fastdbl: then we should implement this in a nice modular way. I'm not sure at the moment if it works to leave out the fastd key repo, as something like that is not needed in the ffruhr setup if I understand it correctly.

json: ok then maybe leave it like that. I thought about working with the json format in the fastd-blacklist.sh file in a nice way. maybe change it to a python script that really parses the format and iterates over the entries. but maybe thats to much effort then...